spec/spec_helper.rb

```diff
@@ -18,21 +19,6 @@
     delete_index
   end
 
-  config.expect_with :rspec do |expectations|
-    expectations.include_chain_clauses_in_custom_matcher_descriptions = true
-  end
-
-  config.mock_with :rspec do |mocks|
-    mocks.verify_partial_doubles = true
-  end
-
-  config.shared_context_metadata_behavior = :apply_to_host_groups
-
-  if config.files_to_run.one?
-    config.default_formatter = "doc"
-  end
-
-  config.profile_examples = 1
```

Comment:
Again, how is this related?

Reply:
Indeed, it's not. I'm trying to feel where the 'cleaning up a file can go in the same PR' line is.